src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@import 'utils/variables';
```

Comment:
Ensure that the 'utils/variables' file contains definitions for the variables '$gap', '$cell-size', and '$days' used in 'calendar.scss'.